<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSearch</class>
 <widget class="QWidget" name="WidgetSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frameSearchSidebar">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_20">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollAreaSearchSidebar">
         <property name="styleSheet">
          <string notr="true">QFrame {
	 background-color: rgb(199, 202, 255);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QFrame" name="frameSearchSidebarContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>586</width>
            <height>516</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_19">
           <property name="margin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="0" column="0">
            <widget class="QToolButton" name="toolButtonNewSearch">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
	background-color: rgb(78, 124, 179);
	color: rgb(255, 255, 255);
	border: none;
	font-size: 16px;
	font-weight: bold;
}

QToolButton:hover {
	background-color: rgb(56, 90, 129);
}</string>
             </property>
             <property name="text">
              <string>New Search</string>
             </property>
             <property name="icon">
              <iconset resource="Resource.qrc">
               <normaloff>:/Resource/Generic/NewSearch.png</normaloff>:/Resource/Generic/NewSearch.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchResultsTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchResultsTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton {
	background-color: rgb(78, 124, 179);
	color: rgb(255, 255, 255);
	border: none;
	font-size: 16px;
	font-weight: bold;
}

QToolButton:hover {
	background-color: rgb(56, 90, 129);
}</string>
               </property>
               <property name="text">
                <string>Results</string>
               </property>
               <property name="icon">
                <iconset resource="Resource.qrc">
                 <normaloff>:/Resource/Network/BrowseStatistics.png</normaloff>:/Resource/Network/BrowseStatistics.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchResultsTask">
               <property name="styleSheet">
                <string notr="true">QFrame {
	background-color: rgb(161, 178, 231);
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_27">
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelSearchResultsSearchingHeader">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Searching:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="labelSearchResultsFoundHeader">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Found:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelSearchResultsSearching">
                  <property name="text">
                   <string>Not Currently Searching</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="labelSearchResultsBlocked">
                  <property name="text">
                   <string>0 Peers, 0 Files</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="labelSearchResultsBlockedHeader">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Blocked:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="labelSearchResultsFound">
                  <property name="text">
                   <string>No Files Found</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchFiletypeTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchFiletypeTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton {
	background-color: rgb(78, 124, 179);
	color: rgb(255, 255, 255);
	border: none;
	font-size: 16px;
	font-weight: bold;
}

QToolButton:hover {
	background-color: rgb(56, 90, 129);
}</string>
               </property>
               <property name="text">
                <string>Any File Type</string>
               </property>
               <property name="icon">
                <iconset resource="Resource.qrc">
                 <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchFiletypeTask">
               <property name="styleSheet">
                <string notr="true">QFrame {
	background-color: rgb(161, 178, 231);
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_25">
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="4" column="0">
                 <layout class="QGridLayout" name="gridLayoutSearchSize">
                  <item row="0" column="0">
                   <widget class="QComboBox" name="comboBoxSearchSizeMin">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="labelSearchSizeTo">
                    <property name="text">
                     <string>to</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QComboBox" name="comboBoxSearchSizeMax">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelSearchSize">
                  <property name="text">
                   <string>File size must be:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QListView" name="listViewSearchFiletype">
                  <property name="styleSheet">
                   <string notr="true">QListView {
	background-color: rgb(255, 255, 255);
}</string>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchNetworksTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchNetworksTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton {
	background-color: rgb(78, 124, 179);
	color: rgb(255, 255, 255);
	border: none;
	font-size: 16px;
	font-weight: bold;
}

QToolButton:hover {
	background-color: rgb(56, 90, 129);
}</string>
               </property>
               <property name="text">
                <string>Networks</string>
               </property>
               <property name="icon">
                <iconset resource="Resource.qrc">
                 <normaloff>:/Resource/Network/Network.png</normaloff>:/Resource/Network/Network.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchNetworksTask">
               <property name="styleSheet">
                <string notr="true">QFrame {
	background-color: rgb(161, 178, 231);
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_23">
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelSearchNetworks">
                  <property name="text">
                   <string>Search on these networks:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <layout class="QGridLayout" name="gridLayoutSearchNetworks">
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="checkBoxSearchGnutella2">
                    <property name="text">
                     <string>Gnutella 2</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/Resource/Networks/Gnutella2.png</normaloff>:/Resource/Networks/Gnutella2.png</iconset>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="checkBoxSearchGnutella1">
                    <property name="text">
                     <string>Gnutella 1</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/Resource/Networks/Gnutella1.png</normaloff>:/Resource/Networks/Gnutella1.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QCheckBox" name="checkBoxSearchAres">
                    <property name="text">
                     <string>Ares</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/Resource/Networks/Ares.png</normaloff>:/Resource/Networks/Ares.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QCheckBox" name="checkBoxSearchEDonkey">
                    <property name="text">
                     <string>eDonkey</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/Resource/Networks/EDonkey.png</normaloff>:/Resource/Networks/EDonkey.png</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton {
	background-color: rgb(78, 124, 179);
	color: rgb(255, 255, 255);
	border: none;
	font-size: 16px;
	font-weight: bold;
}

QToolButton:hover {
	background-color: rgb(56, 90, 129);
}</string>
               </property>
               <property name="text">
                <string>Search</string>
               </property>
               <property name="icon">
                <iconset resource="Resource.qrc">
                 <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchTask">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>160</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QFrame {
	background-color: rgb(161, 178, 231);
}

QToolButton {
    border: 1px solid transparent;
    background-color: transparent;
}

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

QToolButton:checked {
     border: 1px solid rgb(78, 96, 255);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QToolButton:hover {
     border: 1px solid rgb(78, 96, 255);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_21">
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="1" column="0" colspan="2">
                 <widget class="QLabel" name="labelSearch">
                  <property name="text">
                   <string>Type your search here:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLineEdit" name="lineEditSearch"/>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QLabel" name="labelSearchFileType">
                  <property name="text">
                   <string>File Type:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QComboBox" name="comboBoxSearchFileType">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0" colspan="2">
                 <layout class="QGridLayout" name="gridLayoutSearchTaskButtons">
                  <item row="0" column="0">
                   <widget class="QToolButton" name="toolButtonSearch">
                    <property name="text">
                     <string>Search</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
                    </property>
                    <property name="toolButtonStyle">
                     <enum>Qt::ToolButtonTextBesideIcon</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QToolButton" name="toolButtonSearchClear">
                    <property name="text">
                     <string>Clear</string>
                    </property>
                    <property name="icon">
                     <iconset resource="Resource.qrc">
                      <normaloff>:/Resource/Generic/Cancel.png</normaloff>:/Resource/Generic/Cancel.png</iconset>
                    </property>
                    <property name="toolButtonStyle">
                     <enum>Qt::ToolButtonTextBesideIcon</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="10" column="0">
            <spacer name="verticalSpacerSearchSidebar">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frameSearchResults">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutSearchResults">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
